frontend/src/utils/apiRequests.ts

```diff
@@ -0,0 +1,18 @@
+import axios from 'axios';
+
+export async function postRequest(path: string, data: any) {
+
+    return await axios.post(`http://localhost:8080/api/${path}`, data);
+}
+
+export async function getRequest(path: string) {
+    return await axios.get(`http://localhost:8080/api/${path}`);
+}
+
+export async function patchRequest(path: string , data: any) {
+    return await axios.patch(`http://localhost:8080/api/${path}`, data);
+}
+
+export async function deleteRequest(path: string) {
+    return await axios.delete(`http://localhost:8080/api/${path}`);
+}
```

Comment:
Can we define "localhost:8080" somewhere? Not sure how that works in typescript.. (if it's a hassle, just ignore this)

Reply:
yes,  will do!